tensorflow/contrib/distributions/python/kernel_tests/bijectors/reshape_test.py

```diff
@@ -150,6 +151,17 @@ def _testInvalidDimensionsOpError(self, expected_error_message):
       with self.assertRaisesError(expected_error_message):
         sess.run(bijector.forward_event_shape_tensor(shape_in),
                  feed_dict=feed_dict)
+
+  def _testInvalidDimensionsStatic(self, expected_error_message):
+    """Version of _testInvalidDimensionsOpError for errors detected statically
```

Comment:
Args section is missing, does not match arg list or is badly indented.
Expected: expected_error_message
Found: 
Missing: expected_error_message
Unexpected:

Reply:
Added documentation for `expected_error_message` argument.